common/src/main/java/com/wynntils/utils/ObjectUtils.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.utils;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.PipedInputStream;
+import java.io.PipedOutputStream;
+
+public class ObjectUtils {
+    /**
+     * Returns a copy of the object, or null if the object cannot
+     * be serialized.
+     */
+    public static Object copy(Object orig) {
+        Object obj = null;
+        try {
+            // Make a connected pair of piped streams
+            PipedInputStream in = new PipedInputStream();
```

Comment:
Yikes! Can't we use clone instead?

Reply:
Object doesn't implement cloneable by default? If you can figure out a better alternative, please say so, I don't like this that much either.

Reply:
To begin with, your implementation can be replaced with the one in `SerializationUtils#clone`. It does the same (possibly with almost identical code) but we don't have to maintain it. The main restriction here is that the classes need to be serializable.

Another suggestion I got from the interwebz was to use e.g.: 
```
User deepCopy = gson.fromJson(gson.toJson(orig), User.class);
```

I'd prefer that over serialization, if I have to chose. (And if it works okay.)